### ISSUE TITLE
Nested .handle doesn't work as expected

### ISSUE THREAD

**Issue by Zensavona** [jgwhite/ember-sortable#69]
Hi, I'm having some issues with getting `.handle` to work given the following layout (emblem):

```
.row.col-md-9
  = sortable-group tagName="div" onChange='reorderItems' as |group|
    = each sortedModel as |item|
      = sortable-item tagName="div" model=item group=group handle='.handle'
        .col-md-12: .block.block-bordered.block-link-hover3
          table.block-table: tbody: tr
            td.bg-gray-lighter.border-r style='width: 100px;'
              img.pull-left src='https://placeholdit.imgix.net/~text?txt=100%C3%97100&w=100&h=100'
            td
              .h5.push-5-t
                span.text-muted.text-uppercase Caption: &nbsp;
                = item.caption
              .h5.push-5-t
                span.text-muted.text-uppercase First Comment: &nbsp;
                = item.first_comment
            td.bg-gray-lighter.border-l style='width: 20px;'
              span.handle data-item=item &vArr;
```

What's happening is that I can drag the items, but they don't move out of the way for each other or reorder, e.g. if I have two elements and i drag the bottom one up, the top one doesn't move at all.

I did some experimentation and have found that a lot of the time a `<span class="handle">` is not recognised unless it's a top element inside the `{{#sortable-item}}`. Is there a way around this? :smile:

**Comment by jgwhite** [comment]
Certainly sounds like a bug. I’m not sure when I’ll get a chance to look at this, but it would help tremendously if you could submit a PR that adds an example of this behaviour to the [demos page](https://github.com/jgwhite/ember-sortable/blob/master/tests/dummy/app/templates/application.hbs) :D

**Comment by Zensavona** [comment]
Ok, so I tried reproducing it on your demo page and couldn't, so I decided to see what happens if I remove all the styling (css classes) from my markup and see if the bug goes away - sure enough it did, so I progressively readded it all until I found the culprit - it was the `col-md-12` inside the `sortable-item` in my code above. No idea why this was breaking the behaviour whenever the handle was inside it.

Unfortunately I can't post a reproduction of the code on GitHub as the markup is from a ThemeForest theme who's license doesn't permit redistribution. I'm not exactly sure what the fine print has to say about this kind of situation but I'll err on the side of caution...

If you'd like any more info I haven't provided let me know, but I'm closing this as I'm no longer affected by it and it's pretty obscure and hard to reproduce.

Thanks for your fast reply :+1:
